modules/badge/css/README.md

```diff
@@ -0,0 +1,74 @@
+# Canvas Kit CSS Badge
+
+The count badge provides a quantity-based summary with dynamic values.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-badge
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-badge/index.scss';
+```
+
+## Accessibility
+
+A common use case for the count badge is for displaying notifications, and there are several
+accessibility concerns you'll want to keep in mind:
+
+- The button should have an aria-label that updates with the count
+- The elements inside the button should have `aria-hidden`
+- The live region should be outside the button
+- The live region should be visually hidden and only contain text
+
+### Example
+
+```html
+<button type="button" aria-label="Alerts 3 new notifications">
```

Comment:
Should this be "3 new notifications" or "Show 3 new notifications"?

Reply:
This text is from the a11y spec: https://cdpn.io/lefthandev/debug/5e5c051bdde98026a4c49dd583769ed3